server/src/modules/users/domain/value-objects/errors.ts

```diff
@@ -6,14 +6,21 @@ export namespace UserValueObjectErrors {
     }
   }
 
-  export class InvalidPassword {
```

Comment:
Does it make sense to rename error? I understand the separation of the UserPassword class but it seems like everywhere this error is actually thrown the object in question is still what we know as a `password` i.e. it hasn't yet been hashed into a `secret`

Reply:
Good point, this seems to be thrown in `user-password.ts:15` where an invalid value is provided. Perhaps a more appropriate name is `InvalidSecretValue` or we could be more granular with our error handling and have `InvalidUnhashedValue` vs `InvalidHashedValue`. I'll go with the former for now, we can swap the naming and refactor if/when the need arises.